lib/multi-write.ts

```diff
@@ -207,8 +207,16 @@ export async function decompressThenFlash({
 			numBuffers,
 		});
 	} finally {
+		// don't be fooled by the await, `decompressThenFlash` promise is already resolved;
+		// if you cleandup tmp yourself you might get a race condition
 		if (decompressedFilePath) {
-			await fs.unlink(decompressedFilePath);
+			try {
+				await fs.unlink(decompressedFilePath);
+			} catch (error) {
+				if (error.code === 'ENOENT') {
+					console.warn(`File ${decompressedFilePath} was already deleted`);
+				}
```

Comment:
This triggers a eslint error for "unsafe-finally", reading about it, it seems fine as we're in a try-catch, but I'd like your opinion on it

Reply:
Looks fine to me as well. The problem is that `finally` replaces the return value of a `try`/`catch` block, but this can only happen in this case when we're handling an error, so it's not a problem.

BTW you seem to have accidentally committed formatting changes.

Reply:
Indeed, I just ran `npm run prettier` so those files must have been out of style. Anyway that fix shouldn't be here.